docs/user_guides/projects/share_and_access/sharing-a-feature-store.md

```diff
@@ -0,0 +1,90 @@
+# Sharing a Feature Store
+
+## Introduction
+
+In hopsworks it is possible to share feature stores among projects, so even if you have a connection to one project,
+you can retireve a handle to any feature store shared with that project
+
+## UI
+
+### Step 1: Open the project of the feature store that you would like to share on Hopsworks.
+
+In the `Project Settings` navigate to `Shared with other projects` section.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/project/share_with_other_projects.png" alt="Share Project">
+  </figure>
+</p>
+
+### Step 2: Share current feature store   
+
+Click `Share feature store` to bring up the dialog for sharing.
+
+In `Project` section choose feature store project you wish to share feature store with.
+
+In `Access rights` section select the permissions level that the project user members should have on the feature store and click Share.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/project/share_dialog.png" alt="Share Project Dialog">
+  </figure>
+</p>
+
+### Step 3: Accept Invitation
+
+In the project where feature store was shared (step 2) go to `Project Settings` and navigate to `Shared from other projects` section.
+Click `accept`.
+
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/project/accept.png" alt="Accept">
+  </figure>
+</p>
+
+Now shared feature stores and corresponding access rights will be listed under `Shared from other projects` section.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/project/list_of_shared_projects.png" alt="List of shared projects">
+  </figure>
+</p>
+
+## Fetch feature groups from shared feature store and join with feature groups from current feature store   
+
+### Step 1: Get feature store handles 
+
```

Comment:
I think we can add some text around this snippet. Something like: "To access features from a shared feature store you need to first retrieve the handle for the shared feature store. To retrieve the handle use the `get_feature_store()` method and provide the name of the shared feature store."